doc/install_mne_c.rst

```diff
@@ -2,290 +2,162 @@
+or later). Both 32- and 64-bit operating systems are supported; a PowerPC
+version for macOS can be provided upon request. At least 2 GB of memory is
+required, 4 GB or more is recommended. The software requires at least 80 MB of
+disk space. MATLAB is an optional dependency; the free `MATLAB runtime`_ is
+sufficient. If MATLAB is not present, the utilities ``mne_convert_mne_data``,
+``mne_epochs2mat``, ``mne_raw2mat``, and ``mne_simu`` will not work.
 
-The present version number is 2.7.4. The <*rev*> field
-is the SVN revision number at the time this package was created.
-The <*Operating system*> field
-is either Linux or MacOSX. The <*processor*> field
-is either i386 or x86_64. The <*ext*> field
-is 'gz' for compressed tar archive files and 'dmg' for
-Mac OSX disk images.
+For boundary-element model (BEM) mesh generation (see
+:doc:`Creating the BEM meshes <manual/appendix/bem_model>`),
+and for accessing the ``tkmedit`` program from ``mne_analyze`` (see
+:ref:`CACCHCBF`), MNE-C needs access to a working installation of FreeSurfer_,
+including the environment variables ``FREESURFER_HOME``, ``SUBJECTS_DIR``, and
+``SUBJECT``.
 
-Installing from a compressed tar archive
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. admonition:: |apple| macOS
+  :class: note
 
-Go to the directory where you want the software to be installed:
+  For installation on macOS, you also need the `XCode developer tools`_, an X
+  Window System such as XQuartz_, and the netpbm_ library. The recommended way
+  to get netpbm is to install MacPorts_, and then run ``sudo port install
+  netpbm`` in the Terminal app.
 
-.. code-block:: console
+Download and Installation
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    $ cd <dir>
+MNE-C is distributed as either a compressed tar archive (.tar.gz) or a macOS
+disk image (.dmg). The `MNE-C download page`_ requires registration with a
+valid email address.  The current stable version is 2.7.3; "nightly" builds of
+the development version are also available on the download page.
 
-Unpack the tar archive:
+To install from the compressed tar archive, change directory to the desired
+install location, and unpack the software using ``tar``:
 
 .. code-block:: console
 
-    $ tar zxvf <software package>
+    $ cd /path/to/install/location
+    $ tar zxvf <path_to_archive_file>
 
-The name of the software directory under <*dir*> will
-be the same as the package file without the .gz extension.
-
-Installing from a Mac OSX disk image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-- Double click on the disk image file.
-  A window opens with the installer package ( <*name*> .pkg)
-  inside.
-
-- Double click the the package file. The installer starts.
-
-- Follow the instructions in the installer.
-
-.. note::
-
-    The software will be installed to /Applications/ <*name*> by default.
-    If you want another location, select Choose Folder... on the Select a
-    Destination screen in the installer.
```

Comment:
*how to click the "Choose Folder" button during macOS installation to change install location* is general software advice, not specific to MNE